app/javascript/packs/document-capture.jsx

```diff
@@ -21,7 +21,9 @@ const device = {
   isMobile: isCameraCapableMobile(),
 };
 
-loadPolyfills(['fetch']).then(() => {
+document.addEventListener('DOMContentLoaded', async () => {
+  await loadPolyfills(['fetch']);
```

Comment:
Could/should we parallelize the polyfill loading + DOM content loading? Something like....

```js
const domContentPromise = new Promise((resolve, reject) => {
  document.addEventListener('DOMContentLoaded', resolve);
});

Promise.all([domContentPromise, loadPolyfills(['fetch'])]).then(() => {
  // ...
})
```

Reply:
> Could/should we parallelize the polyfill loading + DOM content loading? Something like....

Yeah, we could do that, and makes sense to me 👍